tests/fixtures/models.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, Callable
+
+from composer.devices import Device
+from omegaconf import DictConfig
+from pytest import fixture
+from transformers import PreTrainedTokenizerBase
+
+from llmfoundry.models.hf.hf_causal_lm import ComposerHFCausalLM
+from llmfoundry.models.model_registry import COMPOSER_MODEL_REGISTRY
+from llmfoundry.models.mpt.modeling_mpt import ComposerMPTCausalLM
+from llmfoundry.utils.builders import build_tokenizer
+
+
+def _build_model(config: DictConfig, tokenizer: PreTrainedTokenizerBase,
```

Comment:
I'm not sure the move to device should be contained in this fixture. Code that gets the model from the constructor should not actually assume that the model is moved to the right device. For testing, we often do move it, but I'm wondering if doing it automatically might hide some tricky bugs

Reply:
ah gotcha, let's stay on the safe side and move it out then.